SkyEditor.RomEditor.Rtdx.ConsoleApp/Scripts/Queries/Rtdx/PrintSymbolLists.csx

```diff
@@ -0,0 +1,19 @@
+﻿#load "../../../Stubs/Rtdx.csx"
+
+using System;
+using SkyEditor.RomEditor.Domain.Rtdx.Structures;
+
+void Print(Sir0StringList stringList)
+{
+    for (int i = 0; i < stringList.Entries.Count; i++)
+    {
+        Console.WriteLine($"{i}: {stringList.Entries[i]}");
+    }
+}
+
+System.Console.WriteLine("-------- DungeonMapSymbol");
```

Comment:
Can be simplified to just Console.WriteLine since the System namespace has been imported